src/accept.jl

```diff
@@ -0,0 +1,54 @@
+"""
+Calculates the ratio between the cost of the current program and the cost of the proposed program.
+If the proposed program has smaller cost, returns true.
+Otherwise, returns true with the probability equal to the ratio.
+In any other case, returns false.
+# Arguments
+- `current_cost::Float`: the cost of the current program.
+- `next_cost::Float`: the cost of the proposed program.
+- `temperature::Float`: the temperature; not used.
+"""
+function probabilistic_accept(current_cost, next_cost, temperature)
+    ratio = current_cost / next_cost
```

Comment:
This is not really a mistake, but there is a different more standard way of defining the ratio:

```julia
ratio = next_cost / (current_cost + next_cost)
return rand() <= ratio
```

I would go with this because other people would expect it to be like this. but again, not really a mistake, but there are discrepancies how often a better program is accepted with these two differences.